### PR TITLE
create a register.jsx under the data folder to export register data

### LINKED ISSUE #251: Extract Register Data
Within the register form, there is static data that will change between hackathons and we want an easy interface to work with them 
image

- Create a Register.jsx file under the data folder which holds such static information

### PR DESCRIPTION
**Pull request by twang0323**
image

### CODE REVIEW

File: src/data/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const requirphotography = {
```
Comment (shahdivyank) [inline src/data/Register.jsx]: create a single requirement array and have an array of objects, reference https://github.com/acm-ucr/hackathon-website/blob/dev/src/data/Volunteers.js#L5

Comment (twang0323) [reply src/data/Register.jsx]: image
like this?

File: src/data/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const Registration = {
```
Comment (shahdivyank) [inline src/data/Register.jsx]: make sure to import it and use it as well

Comment (twang0323) [reply src/data/Register.jsx]: image
I imported it into form/register and used it in the second line

Comment (shahdivyank) [reply src/data/Register.jsx]: make sure to push these changes as well, doesn't look like they are pushed

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**